packages/web-components/fast-components/src/design-system-provider/index.ts

```diff
@@ -153,6 +153,108 @@ export class FASTDesignSystemProvider extends DesignSystemProvider
     })
     public disabledOpacity: number;
 
+    @attr({ ...fromView, attribute: "type-ramp-minus-2-size" })
+    @designSystemProperty({
+        cssCustomProperty: "type-ramp-minus-2-size",
+        default: "10px",
+    })
+    public typeRampMinus2Size: string;
+
+    @attr({ ...fromView, attribute: "type-ramp-minus-2-height" })
```

Comment:
@bheston @nicholasrice do we want `line-height` here or just height? How important are the semantics of the `@attr`?

Reply:
If we do that I'd say we should also use "font-size" instead of "size" for consistency. I'm good with it, but curious on @bheston's feedback as this was the name that was outlined in the spec.

Reply:
I agree there - I don't think it's blocking but we use height elsewhere and it is a different property altogether. Similar with "size" to your point.

Reply:
I see that point, and considered it when writing the spec. I could go for being more explicit, but I think I was trying to keep it shorter and more generic than actual attribute names. I can't think of a way where being more explicit hurts us though, so maybe that's best. I just double-checked some of the other typography taxonomy work that's going on and they seem to be leaning toward full attribute names, so let's make that change. 👍